### ISSUE TITLE
Log spam in HA 0.111.0

### ISSUE THREAD

**Issue by Marlor** [py-smart-gardena/hass-gardena-smart-system#29]
After an upgrade to HA 0.111.0 the home-assistant.log gets filled with:
```
2020-06-10 20:57:05 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:57:05 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 20:57:34 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:57:34 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 20:58:05 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:58:05 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 20:58:05 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:58:05 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 20:59:04 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:59:04 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 20:59:35 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 20:59:35 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 21:00:08 INFO (Thread-6) [gardena.smart_system] Received COMMON message
2020-06-10 21:00:35 INFO (Thread-6) [gardena.smart_system] Received MOWER message
2020-06-10 21:00:35 INFO (Thread-6) [gardena.smart_system] Received COMMON message
```

**Comment by grm** [comment]
Hum,

Did you activate the logging in Home assistant for the plugin ?

Grm

**Comment by Marlor** [comment]
No but even without any config, the logging level defaults to "debug" for every plugin:
https://www.home-assistant.io/integrations/logger/

**Comment by grm** [comment]
Ok, I was not aware that debug was the default level for all plugins. We will see what can be done about that  !

**Comment by evadue** [comment]
This happens to me also. Would be great if you could fix it!

**Comment by grm** [comment]
This should be fixed in https://github.com/py-smart-gardena/hass-gardena-smart-system/releases/tag/0.2.0b2 and available in 0.2.0
